feature-rich/index.ts

```diff
@@ -487,513 +511,507 @@ const EMOJIS_ARRAY = [
-		ImageUpload,
-		Base64UploadAdapter,
-		ImportWord,
-		Indent,
-		IndentBlock,
-		Italic,
-		Link,
-		LinkImage,
-		List,
-		ListProperties,
-		MediaEmbed,
-		Mention,
-		PageBreak,
-		Paragraph,
-		PasteFromOffice,
-		PictureEditing,
-		RemoveFormat,
-		SpecialCharacters,
-		SpecialCharactersEmoji,
-		SpecialCharactersEssentials,
-		Strikethrough,
-		Style,
-		Subscript,
-		Superscript,
-		Table,
-		TableCaption,
-		TableCellProperties,
-		TableColumnResize,
-		TableProperties,
-		TableToolbar,
-		TextPartLanguage,
-		TextTransformation,
-		TodoList,
-		Underline,
-		UploadAdapter,
-		WordCount,
-		WProofreader,
-		// SlashCommand,
-		// Template,
-		// FormatPainter,
-		// TableOfContents,
-	],
-	licenseKey: PRODUCTIVITY_PACK_LICENSE_KEY,
-	toolbar: {
-		shouldNotGroupWhenFull: true,
-		items: [
-			// --- Document-wide tools ----------------------------------------------------------------------
-			'undo',
-			'redo',
-			'|',
-			'importWord',
-			'exportWord',
-			'exportPdf',
-			'|',
-			'findAndReplace',
-			'selectAll',
-			'wproofreader',
-			'|',
-			// 'insertTemplate',
-			// 'tableOfContents',
-			// '|',
-			// 'formatPainter',
-			// '|',
+ClassicEditor.create(
+	document.querySelector('#cke5-feature-rich-demo') as HTMLElement,
+	{
+		plugins: [
+			Alignment,
+			Autoformat,
+			AutoImage,
+			AutoLink,
+			BlockQuote,
+			Bold,
+			...(CKBOX_TOKEN_URL ? [CKBox] : []),
+			CloudServices,
+			Code,
+			CodeBlock,
+			Essentials,
+			FindAndReplace,
+			Font,
+			GeneralHtmlSupport,
+			Heading,
+			Highlight,
+			HorizontalLine,
+			Image,
+			ImageCaption,
+			ImageInsert,
+			ImageResize,
+			ImageStyle,
+			ImageToolbar,
+			ImageUpload,
+			Base64UploadAdapter,
+			Indent,
+			IndentBlock,
+			Italic,
+			Link,
+			LinkImage,
+			List,
+			ListProperties,
+			MediaEmbed,
+			Mention,
+			PageBreak,
+			Paragraph,
+			PasteFromOffice,
+			PictureEditing,
+			RemoveFormat,
+			SpecialCharacters,
+			SpecialCharactersEmoji,
+			SpecialCharactersEssentials,
+			Strikethrough,
+			Style,
+			Subscript,
+			Superscript,
+			Table,
+			TableCaption,
+			TableCellProperties,
+			TableColumnResize,
+			TableProperties,
+			TableToolbar,
+			TextPartLanguage,
+			TextTransformation,
+			TodoList,
+			Underline,
+			WordCount,
+			// @TODO WProofreader needs to be migrated to NIM compatible package first to work here.
+			// ...(WEB_SPELL_CHECKER_LICENSE_KEY ? [WProofreader] : []),
+			...(LICENSE_KEY ? [
+				ExportPdf,
+				ExportWord,
+				ImportWord,
+				SlashCommand,
+				Template,
+				FormatPainter,
+				TableOfContents,
+			] : []),
+		],
+		licenseKey: LICENSE_KEY,
+		toolbar: {
+			shouldNotGroupWhenFull: true,
+			items: [
+				// --- Document-wide tools ----------------------------------------------------------------------
+				'undo',
+				'redo',
+				'|',
+				'importWord',
+				'exportWord',
+				'exportPdf',
+				'|',
+				'findAndReplace',
+				'selectAll',
+				'wproofreader',
+				'|',
+				// 'insertTemplate',
```

Comment:
I've just tripped on this. I provided the license key, as it asked me to, but then surprise, surprise: those premium buttons weren't there. I had quite a big WTH moment :D 

The good thing about our toolbar initialization is that even if you used a button that wasn't registered, the editor will work and you will get a warning.  
  
So it's not necessary to comment out missing buttons. And it's definitely confusing :P